pkg/webhook/kapiserver/ensurer.go

```diff
@@ -48,7 +47,7 @@ type ensurer struct {
 // exposed for testing
 var NewSecretsManager = extensionssecretsmanager.SecretsManagerForCluster
 
-const fakeTokenSecretName = constants.ApplicationName + "-fake-token"
+const FakeTokenSecretName = constants.ApplicationName + "-fake-token"
```

Comment:
As this constant is no longer used in the current package, you can move it to `pkg/controller/lifecycle/actuator.go` and unexport it again.